packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1690,3 +1690,15 @@ oie.enrollment.policy.grace.period.required.in.one.day = Required in 1 day
 
 # OAMP Strings
 api.policy.okta.account.management.insufficient.authenticators.unable.to.sign.in = Unable to sign in. Contact support for assistance.
+
+# Chrome Local Network Access
+chrome.lna.fast.pass.requires.permission.title = Okta FastPass requires network permission
```

Comment:
nit: I think the period seperation in `fast.pass` is unnecessary, and should just be `fastpass`